docs/website/docs/user-guides/quickstart/java.md

```diff
@@ -12,10 +12,15 @@ import InitialSourceCodeInfo from './docs-mdx/initial_source_code_description.md
 For Java, we will use the [Spring Initializr](https://start.spring.io/) to generate the example source code:
 
 1. Navigate to [start.spring.io](https://start.spring.io/) 
-2. Select **11** under **Java**
-3. Click on "Add" under "Dependencies"
-4. Select "Spring Web"
-5. Click "Generate" to generate and download the source code
+2. Under **Spring Boot** select **2.7.\*** version.
```

Comment:
Can you move this to below 2? I think it will be more relevant there.
```suggestion
2. Under **Spring Boot** select **2.7.\*** version.
:::caution
For now, don't use SpringBoot 3. Currently there is no Devfile in [registry.devfile.io](https://registry.devfile.io/) with Java 17.
:::
```

Reply:
ok, I thought about it it looked a bit weird to me to break the list with the note. But now at the second look it doesn't look that bad.

Reply:
/hold

Can we remove the previous note then (after the list)? It currently appears twice.

Reply:
![image](https://user-images.githubusercontent.com/593208/210782656-0360ad09-54ae-4644-ba86-ea53f4ce7a30.png)

Reply:
Whoops - my comment was sent too late :)